Cargo.lock

```diff
@@ -3466,6 +3478,21 @@ dependencies = [
  "panic-semihosting",
 ]
 
+[[package]]
+name = "tests-gimletlet"
+version = "0.1.0"
+dependencies = [
+ "build-util",
+ "cfg-if",
+ "cortex-m",
+ "cortex-m-rt",
+ "kern",
+ "panic-halt",
+ "panic-itm",
+ "panic-semihosting",
+ "stm32h7 0.13.0",
```

Comment:
Should this be updated to `0.14.0`? It looks like it's the only crate pulling in `0.13.0`.

Reply:
possibly! I tried to change as little as possible, but that does seem like a good idea.

Reply:
(yes, fwiw.)